lib/src/Utilities.cc

```diff
@@ -957,7 +983,9 @@ trantor::Date getHttpDate(const std::string &httpFullDateString)
         // weird RFC 850-hybrid thing that reddit uses
         "%a, %d-%b-%Y %H:%M:%S",
     };
-    struct tm tmptm;
+    struct tm tmptm
+    {
+    };
     for (const char *format : formats)
```

Comment:
Just declare a variable, do we need to modify it like this?

Reply:
Again, it's just a clang-tidy warning I might have fixed while looking at the file. Sorry for this